doc/src/BCMPackage.rst

```diff
@@ -10,6 +10,11 @@ bcm_find_package
 
 This works the same as cmake's ``find_package`` except in addition, it will keep track of the each call to ``bcm_find_package`` in the project. Functions like ``bcm_package`` and ``bcm_boost_package`` will include these dependencies automatically in cmake's package config that gets generated.
 
+.. I don't understand why 'bcm_find_package' needs to do anything other than
+   what the standard find_package does. Can't the targets exported by
+   'find_package' carry all the "package config" information needed for '.pg'
+   file generation?
+
```

Comment:
Also, the `PACKAGE_FOUND` property could be used for this, however, this is a global property, so it will break superproject builds, I believe. Let me test this.

It could possible be named differently as well, since its only to be used for non-build dependencies.